src/SalesComparison.cpp

```diff
@@ -473,3 +473,31 @@ double SalesComparison::compareYesterday() {
         return 0;
     }
 }
+
+void SalesComparison::suggestSale() {
+    double worst_perc;
+    unsigned long worst_id;
+    std::string ret;
+    std::map<unsigned long, double>::iterator it;
+
+    worst_perc = 1000000000000000000000.0;
+    worst_id = -1;
+    for (it = currentMonthItemIds.begin(); it != currentMonthItemIds.end(); it++) {
+        if (it->second / (itemIdsByMonth[curr_m][it->first] * (1 - daysLeftMonth)) < worst_perc) {
+            // printf("id : %lu     total : %f\n", it->first, it->second);
```

Comment:
If someone else comments changes, I'd like for these to be removed, but don't no need to push changes just to remove these.